docs/src/pages/start/contribution-guide.md

```diff
@@ -3,4 +3,276 @@ title: Contribution Guide
+```
+
+#### Install dependencies
+
+Install the required dependencies.
+```bash
+$ cd quasar
+$ yarn # or npm install
+```
+
+#### Running documentation against your local repository
+
+```bash
+$ quasar dev
+```
+
+The documentation runs against your local cloned repository.
+
+#### Edit and test your changes locally
+
+#### Commit your changes
+
+When you're happy with the change  on your computer, you need to commit the changes to Git:
+```bash
+$ git commit -a
+```
+
+This should fire up your editor to write a commit message. When you have finished, save and close to continue.
+
+#### Update your local repository
+
+It's pretty likely that other changes to master have happened while you were working. Go get them.
+
+1. Add a remote Quasar repository as an upstream
+```bash
+$ git remote add upstream https://github.com/quasarframework/quasar.git
+```
+
+2. Check out your fork's local `dev` branch.
+
+```bash
+$ git checkout dev
+> Switched to branch 'dev'
+```
+
+3. Merge the changes from `upstream/dev` into your local `dev` branch. This brings your fork's `dev` branch into sync with the upstream repository, without losing your local changes.
+
+```bash
+$ git merge upstream/dev
+```
+
+No conflicts? Tests still pass? Change still seems reasonable to you? Then move on and  open a pull request to apply your changes to the dev branch in main Quasar repository.
+
+#### Issue a pull request
+
+Navigate to your repository you just pushed to (e.g. https://github.com/your-user-name/quasar) and click on "New Pull Request" seen in the left top panel.
+
+Ensure the changesets you introduced are included. Fill in some details about your potential patch including a meaningful title. When finished, press Click on "Create pull request". The Quasar core team will be notified about your submission.
+
+## Create new Quasar App Extension
+
+An easy way to start to contribute to Quasar is to generalize the code you have created on your project and publish it as a Quasar App Extension. Follow this guide on how to [create a new extension](../app-extensions/development-guide/introduction).
+
+When done, submit a PR on [Quasar Awesome](https://github.com/quasarframework/quasar-awesome/blob/master/README.md#community-app-extensions), share your achievement via [Quasar Forum](https://forum.quasar-framework.org/category/15/v1-app-extensions). After the review we will [publish your extension](../app-extensions/discover).
+
+## Contributing to the Quasar Core Source Code
+
+As with any project, there are rules to contributing. Ours are written here, please read them carefully. After that, read the [Quasar code of conduct](https://github.com/quasarframework/quasar/blob/dev/.github/CODE_OF_CONDUCT.md) and you’ll be ready to contribute to Quasars’s core repositories.
+
+An article [Look at the source code](https://medium.com/quasar-framework/wip-look-at-the-source-code-please-1b905ea4906) will help you to get familiar with a Quasar code base.
```

Comment:
@smolinari - is this up to date with 1.0?

Reply:
It's using the older version. I'll update it.

Scott

Reply:
Updated!!!

Scott